com.w1hkj.fldigi.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
```

Comment:
Feel free to also add a desktop file for Flarq. Flatpak supports multiple desktop files for one Flatpak. For example, see OpenMW.

Reply:
By the way, if desktop files are available in the tar.gz archive, you can use them for there. You don't need to include them in the Flatpak Git repo.

Reply:
this is not clear to me
if the program brings the .desktop
so I shouldn't recreate it
or you have to specify something in the manifest

Reply:
fldigi has no icon in png usa xpm
I tried several ways to do it but it didn't work
installing on a local pc
fldigi andava but did not show the icon
so how are you at the moment it's fine